### PR TITLE
Fill Patch dashboard card with the real data 

### PR DESCRIPTION
**Pull request by jiridostal**
Wire the real data to Patch card in the dashboard

## Screenshots

image

### CODE REVIEW

File: src/SmartComponents/PatchManager/PatchManagerCard.js [hunk 1]
```diff
@@ -1,51 +1,106 @@
...
-                colorScale={ colorScale }
-            />
+            {!isLoaded ? <Loading/> :
+                <React.Fragment>
+                    <Button
+                        component="a"
+                        href={ `${UI_BASE}/${PATCHMAN_ID}/systems` }
+                        variant="link"
+                        isInline
+                        style={ { textAlign: 'left', zIndex: 3 } }
```
Comment (christiemolloy) [inline src/SmartComponents/PatchManager/PatchManagerCard.js]: why does this need a z-index of 3?

Comment (AllenBW) [reply src/SmartComponents/PatchManager/PatchManagerCard.js]: re this, requesting that we apply the styles to a class defined in a colocated .scss

Comment (jiridostal) [reply src/SmartComponents/PatchManager/PatchManagerCard.js]: I forgot to mention this one... For some reason, the link is not hoverable without z-index override but honestly I didn't dig very deep to find out what causes it. I blame PF Pie Chart which has this in it 
image

Comment (christiemolloy) [reply src/SmartComponents/PatchManager/PatchManagerCard.js]: I actually just updated the pie chart in this PR: https://github.com/RedHatInsights/insights-dashboard/pull/72 and now the link will work without the z-index

Screen Shot 2020-03-23 at 3 55 07 PM

Comment (jiridostal) [reply src/SmartComponents/PatchManager/PatchManagerCard.js]: Removed z-index and moved the style to the separate file.

**Comment by AllenBW** [comment]
@jiridostal can yah resolve those conflicts so we can merge this 🎨 👩‍🎨 🤸‍♀ 🙏

**Comment by AllenBW** [comment]
travis passed n reporting was stuck, we ain't got time for it to sort itself 🧟 https://travis-ci.org/github/RedHatInsights/insights-dashboard/builds/666331414

**Pull request merged**